ecs_composex/elbv2/elbv2_ecs.py

```diff
@@ -261,57 +261,6 @@ def set_new_resource_outputs(self, output_definition):
         return value
 
 
-def validate_tcp_health_counts(props):
```

Comment:
Why are we removing this function? Is it totally unused and I should have deleted it long time ago?

Reply:
https://github.com/compose-x/ecs_composex/blob/main/ecs_composex/elbv2/elbv2_ecs.py#L312

`validate_tcp_health_counts` is only ever used inside function `fix_nlb_settings` which we're also removing below

Reply:
Have you tried then to have `HealthCheckIntervalSeconds` set 6 or 7 then and did it work? (by work, I mean successful deployment to AWS via CFN)
The point of these validations / fix functions is to catch incorrect input and correct it with best effort.

I must have missed that news on the change of health check possible for NLBs/ALBs.

Reply:
@louismollick any insignts on this? did you try creating the NLB with invalid interval values for example? if so, what happened?
thank you

Reply:
@JohnPreston Hey! Sorry for the radio silence, I got sidetracked

I edited and deployed the nginx example from your repo to show the NLB values: 
https://github.com/compose-x/ecs_composex/compare/main...louismollick:ecs_composex:elbv2ParamsFixDemo 
```
healthcheck: 80:TCP:2:10:300:120
```
^^ here the `HealthCheckIntervalSeconds` is `300` and `HealthCheckTimeoutSeconds` is `120`

I deployed from this branch using `poetry run ecs-compose-x up -f docker-compose.yaml -f aws-compose-x.yaml -n testNLB`

Below are some screenshots from the results in AWS, it even shows these values when you go to edit them in Target Group healthcheck settings:
<img width="1120" alt="Screen Shot 2024-01-18 at 11 40 09 PM" src="https://github.com/compose-x/ecs_composex/assets/40916893/48e54e95-ecb5-4216-8059-a124a653eea1">
<img width="667" alt="Screen Shot 2024-01-18 at 11 40 21 PM" src="https://github.com/compose-x/ecs_composex/assets/40916893/54db67c2-c090-422a-a9cf-7acc504c38f0">

Also attached the template here:
[template.json](https://github.com/compose-x/ecs_composex/files/13984422/template.json)

Reply:
Thanks so much. There I thought I was keeping up with all the changes getting the announcement news but there we go, that's some nice changes AWS did!

Thanks @louismollick